client/components/SendZoopDialog.tsx

```diff
@@ -0,0 +1,155 @@
+                })
+                .unwrap()
+                .then((payload) => {
+                    console.log('fulfilled', payload);
+                    setLastZoopSentId(payload.zoop.id)
+                })
+                .catch((error) => console.error('rejected', error));
+              
+                setZoopSent(true)
+            } else {
+                throw new Error('No such user exists. Please select a different username');
+                // TODO: How best to render this error in UI?
+            }
+            
+        }
+        
+    }
+
+    return ( //TODO: MUI suggests adding a wrapper element so that Tooltip works when Fab is disabled
+        <>
+            <Tooltip 
+                title="Send a Zoop"
+            >
+                <Fab
+                    disabled={!token}
+                    sx={{
+                        position: "absolute",
+                        bottom: 25,
+                        right: 25
+                    }}
+                    onClick={() => setOpen(true)}
+                > <SendIcon />
+                </Fab>
+            </Tooltip>
+
+            {/* Render a success dialog if Zoop was sent and successfully integrated into DB,
+                or an input dialog if Zoop has not yet been sent */}
+            {zoopSent ? (
+                <Dialog
+                    open={open}
+                    onClose={() => setOpen(false)}
+                >
+                    <DialogTitle>Zoop Sent!</DialogTitle>
+                    <DialogContent>
+                        <DialogContentText> 
+                            <Link
+                                component="button"
+                                onClick={() => navigate(`/zoops/${lastZoopSentId}`)}
+                            > 
+                                Go to this page to view your Zoop.
+                            </Link>
+                        </DialogContentText>
+
+                    </DialogContent>
+                </Dialog>
+            ) : (
+                <Dialog
+                    open={open}
+                    onClose={() => setOpen(false)}
+                >
+                    <DialogTitle>Compose a Zoop</DialogTitle>
+                    <DialogContent>
+                        <TextField
+                            autoFocus
+                            required
+                            margin="dense"
+                            label="Recipient"
+                            type="text"
+                            placeholder="Username"
+                            onChange={(e) => setUsername(e.target.value)}
+                            value={username}    
+                        />
+                        <TextField
+                            required
+                            fullWidth
+                            multiline
+                            minRows={5}
+                            margin="dense"
+                            label="Content"
```

Comment:
Could we make the label a little more fun to match the Zoop branding? eg. "Say something" or "What do you want to say?"